### DISCUSSION TITLE
Why not add fabric?

### DISCUSSION THREAD

**Discussion by MikeyBeez** [open-webui/open-webui#4111] [General]
Fabric is a great way to curate system messages

**Describe the solution you'd like**
I'd like to see some of the features in my RAGAgent repo added includinf fabric.  https://github.com/MikeyBeez/RAGAgent.

**Comment by CrazyWolf13** [comment]
Fabric would really be awesome!

@tjbck is there anything blocking an easy integration?


Edit: Here the original Fabric link, possibly that makes more sense: https://github.com/danielmiessler/fabric

**Comment by Felix-Pedro** [comment]
Is this being/already been implemented? If not, what happened?
It would be very useful =)
